public/pages/Rules/containers/Rules/Rules.tsx

```diff
@@ -120,7 +116,7 @@ export const Rules: React.FC<RulesProps> = (props) => {
         </EuiFlexItem>
         <EuiFlexItem>
           <EuiPanel>
-            <RulesTable loading={loading} ruleItems={ruleItems} showRuleDetails={setFlyoutData} />
+            <RulesTable loading={loading} ruleItems={allRules} showRuleDetails={setFlyoutData} />
```

Comment:
Is `allRules` using same interface as the `ruleItems`?

Reply:
Yes, just a naming consistency, as here we get all the rules, pre-packaged and custom once, hence changed the naming to **allRules**